mappings/net/minecraft/util/collection/LinkedBlockPosHashSet.mapping

```diff
@@ -0,0 +1,92 @@
+CLASS net/minecraft/class_6136 net/minecraft/util/collection/LinkedBlockPosHashSet
+	COMMENT Represents a set of block positions (long representation).
+	COMMENT <p>
+	COMMENT Uses a {@link Long2LongLinkedOpenHashMap} as its internal storage medium
+	COMMENT to facilitate the quick addition and removal of block positions.
+	COMMENT <p>
+	COMMENT Positions are index into a 2x cubed area that then stores as a long, a bitset
+	COMMENT representing which positions within that area are currently set.
+	COMMENT <p>
+	COMMENT This has two major advantages:
+	COMMENT <ol>
+	COMMENT <li>Positions that are geometrically close together are grouped together in memory. This localises adjacent reads and writes.</li>
+	COMMENT <li>A larger number of positions can be comprised together into one long allowing for a smaller memory footprint.</li>
+	COMMENT </ol>
+	COMMENT @see LevelPropagator
+	FIELD field_31715 buffer Lnet/minecraft/class_6136$class_6137;
+	METHOD <init> (IF)V
+		ARG 1 expectedSize
+		ARG 2 loadFactor
+	METHOD add (J)Z
+		COMMENT Marks a block position as "set".
+		ARG 1 posLong
+	METHOD isEmpty ()Z
+		COMMENT Checks whether there are any block positions that have been "set".
+		COMMENT
+		COMMENT @return {@code true} is this collection is empty.
+	METHOD rem (J)Z
+		COMMENT Marks a block position as "not set". Effectively removing it from this collection.
+		ARG 1 posLong
+	METHOD removeFirstLong ()J
+		COMMENT  Pops first block position off of this set.
+	METHOD size ()I
+		COMMENT @throws UnsupportedOperationException
+	CLASS class_6137 Storage
+		COMMENT Represents a three-dimensional mapping from a block position to a bitset
+		COMMENT of values set at that position.
+		FIELD field_31716 STARTING_OFFSET I
+		FIELD field_31717 HOR_COLUMN_BIT_SEPARATION I
```

Comment:
`Hor`?

Reply:
Short for HORIZONTAL.

Reply:
Basically, it's half the width of the middle field.

First field is `start --> (MID - HOR)`
Second field is `(MID - HOR) ---> (MID + HOR)`
Third field is `(MID + HOR) ---> end`

At least as best I can remember.

Reply:
Should just use `HORIZONTAL_COLUMN_BIT_SEPARATION` instead.

Reply:
It'll be too long tho >.>

Reply:
Done